### ISSUE TITLE
Attempting refresh of monitored enclosure results in exception indicating not supported.

### ISSUE THREAD

**Issue by ChrisLynchHPE** [HewlettPackard/POSH-HPOneView#305]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
Perform a refresh of a monitored enclosure.

### Actual Behavior
```PowerShell
Get-HPOVEnclosure -Name MonitoredEncl | Update-HPOVEnclosure -Refresh

Update-HPOVEnclosure : Updating the enclosure configuration is not supported for a monitored enclosure. Do not specify configuration for a monitored enclosure.
At line:1 char:63
+ ...enclosure -Name MonitoredEncl  | Update-HPOVEnclosure -Refresh
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Send-HPOVRequest:String) [Update-HPOVEnclosure], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperation,Update-HPOVEnclosure
```

**Comment by VladislavSavchenko** [comment]
Many thanks, very much waiting!

**Comment by ChrisLynchHPE** [comment]
This was addressed in the [Release 3.10.1588.1229 ](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.10.1588.1229).

Closing.
